e2e/node/e2e.test.ts

```diff
@@ -250,6 +250,39 @@ describe(`End-to-end access grant tests for environment [${environment}}]`, () =
       ]);
     });
 
+    // The following test is disabled until ESS adds support for recursive Access Grants.
+    // eslint-disable-next-line jest/no-disabled-tests
+    it.skip("can issue a non-recursive access grant", async () => {
```

Comment:
for my own knowledge: what environment do you point at to test these skipped tests?

Reply:
It'd be dev-next, but we really need a 2.x-preprod test environment, which I don't think we currently have.

Reply:
That's because dev-next is actually more closely a 2.1 environment

Reply:
When the ESS team has implemented recursive access grants support, it will get deployed to dev-next, at which point we'll be able to conditionally skip this test instead of systematically skipping it.

Reply:
so this is true TDD then @NSeydoux since these features arent currently in dev-next to be tested?